docs/attributes-registry/db.md

```diff
@@ -34,18 +34,13 @@ This group defines the attributes used to describe telemetry in the context of d
 
 **[1]:** It is RECOMMENDED to capture the value as provided by the application without attempting to do any case normalization.
 
-A single database query may involve multiple collections.
-
-If the collection name is parsed from the query text, it SHOULD only be captured for queries that
-contain a single collection and it SHOULD match the value provided in
-the query text including any schema and database name prefix.
+The collection name SHOULD NOT be extracted from `db.query.text`,
+unless the query format is known to only ever have a single collection name present
+and the collection name is known to always be under the captured `db.namespace`.
```

Comment:
> collection name is known to always be under the captured `db.namespace`

curious - why?

Also maybe 

```suggestion
and the collection name is known to always belong to the captured `db.namespace`.
```

(I've read it first as that collection name should be captured as `db.namespace`)

Reply:
> > collection name is known to always be under the captured `db.namespace`
> 
> curious - why?

I was thinking in cases where there's only ever a single collection name that it would probably(?) be easy to ensure (and would be a nice normalization), but I've removed this part from the PR since it's tangential anyways and I'll open separate issue to explore.